test/rulesets/LinearAlgebra/norm.jl

```diff
@@ -8,6 +10,7 @@
         ),
         T in (Float64, ComplexF64),
         sz in [(3,), (3, 3), (3, 2, 1)]
+println("starting unexported fnorm=$fnorm, T=$T, sz=$sz")
```

Comment:
This needs to be removed before we can merge
```suggestion
```

Reply:
Yes. But figuring out which tests are broken is pretty crazy sometimes. I wonder if it would be worth some work to make this easier.